### PR TITLE
Add followedContents to fetch followed artists in a show

### PR DESCRIPTION
**Pull request by ashkan18**
# Depends on
https://github.com/artsy/gravity/pull/12185

# Change
Add `followedContents` to `show` type so we can get user's followed artists in a specific show.

We can wait till https://github.com/artsy/gravity/pull/12185 gets into prod to merge this.

# Example Query
```graphql
show(id:'something') {
  followedArtists{
    id
    edges {
      node{
        artist{
          id
        }
      }
    }
  }
}
```

### CODE REVIEW

File: src/schema/show.ts [hunk 1]
```diff
@@ -43,6 +43,15 @@ import EventStatus from "./input_fields/event_status"
 import { LOCAL_DISCOVERY_RADIUS_KM } from "./city/constants"
 import { ResolverContext } from "types/graphql"
 
+const ShowsFollowedContentType = new GraphQLObjectType<any, ResolverContext>({
+  name: "ShowsFollowedContent",
+  fields: () => ({
+    artists: {
+      type: new GraphQLList(Artist.type),
```
Comment (mzikherman) [inline src/schema/show.ts]: should this be a connection?

Comment (ashkan18) [reply src/schema/show.ts]: i pretty much followed the current implementation in Fair 
https://github.com/artsy/metaphysics/blob/1c1145abd24a3e2c572d672a88dab57d50d4d94f/src/schema/fair.ts#L38
but we can make it a connection here, what do you think?

Comment (mzikherman) [reply src/schema/show.ts]: I don't mean to intrude or do 'drive by' code reviews (and am mindful of a rush for LD work). Connections have advantages over simple lists for several reasons: there's a schema for additional metadata (like pagination info, total counts, etc- which all frequently wind up being needed by the front-end at some point). Also, there are several Relay capabilities (used in Emission, pagination containers, etc.)- that essentially require a connection type and something conforming to that spec.

So it's just an automatic reflex- if I see a list/array of objects like this, I try to ask if it should be a connection! 😄 Totally up to you though.

Comment (ashkan18) [reply src/schema/show.ts]: makes total sense, let me see if i can quickly add it 🙌

Comment (alloy) [reply src/schema/show.ts]: @mzikherman Here’s a doc to point to for your next drive-by https://github.com/artsy/README/blob/master/playbooks/graphql-schema-design.md#connections-over-lists (and please do expand on it with your favourite arguments)

**Comment by ashkan18** [comment]
Gravity part of this is deployed this should be 👌 to go 🙌

**Comment by anandaroop** [comment]
Thanks 🎉

**Pull request merged**